library/src/tokens/types.ts

```diff
@@ -22,7 +22,7 @@ interface TokenEncryption {
 interface Token {
   id: string;
   tenantId: string;
-  type?: TokenType;
+  type: TokenType;
```

Comment:
@brigonzalez @lcschy @djejaquino is there a way to have a default value here? We currently default to `"token"` in the dotnet SDK, but I couldn't find a way to supply default values with TS interfaces/types.

Reply:
You can't set a default value in interfaces. You could make it optional in the `CreateTokenModel` and set the default value in the implementation.